openedx-unsupported/edx-documentation
# docs: added Edx Discovery API docs

## [ENT-4877](https://openedx.atlassian.net/browse/ENT-4877)

Added documentation for EdX discovery API, specifically for taxonomy.


### Testing

-  clone this repo.
-  create new virtual env
-  install requirement by `make requirements`
-  install more requirements by `pip install -r shared/tools.txt`
-  run server by `./develop.sh en_us/discovery_api/`
- open `http://127.0.0.1:9090` to see docs

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Partner support: 
- [ ] PM review: 

FYI: Tag anyone else who might be interested in this PR here.


### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
